### ISSUE TITLE
Add dirty count for repo

### ISSUE THREAD

**Issue by maplepy** [jorgebucaran/hydro#41]
What i mean by that, is to have a number following or preceding the symbol for a dirty repo. Kinda like tide does it 
![image](https://user-images.githubusercontent.com/28269333/214273228-0b65fb51-b3d9-4dcd-b29a-955f4fce5330.png)

I really love hydro but this is one of the feature missing for me :(

**Comment by jorgebucaran** [comment]
Maybe a flag to show or hide this number?

**Comment by maplepy** [comment]
Sure ! would love it!

**Comment by ankurkotwal** [comment]
I've submitted PR #44 for this issue. I didn't make the show/hide configurable since the ahead/behind numbers aren't configurable either.

**Comment by jorgebucaran** [comment]
No feature flag is a deal breaker for me as I don't intend to use this myself.

**Comment by ankurkotwal** [comment]
Fair enough. I've pushed another commit to the PR adding the feature flag. Does this suffice?

**Comment by ankurkotwal** [comment]
@jorgebucaran is there anything more you need for this PR?

**Comment by jorgebucaran** [comment]
See you at #44.
